src/app/src/pages/model-audit/components/AdvancedOptionsDialog.tsx

```diff
@@ -119,18 +120,25 @@ export default function AdvancedOptionsDialog({
             <Typography variant="subtitle1" fontWeight={600} gutterBottom>
               Scan Timeout
             </Typography>
-            <TextField
+            <BaseNumberInput
               fullWidth
-              type="number"
               value={localOptions.timeout}
-              onChange={(e) =>
+              onChange={(v) =>
                 setLocalOptions({
                   ...localOptions,
-                  timeout: Number.parseInt(e.target.value) || 3600,
+                  // @ts-expect-error - undefined will be clamped to 3600 onBlur.  This approach resolves issue where user cannot clear the field.  This would be better addressed using a more robust form approach and setting default on submit.
+                  timeout: v,
```

Comment:
why not handle undefined in onChange
```suggestion
                  timeout: v ?? 3600,
```